nexus-writer/src/main.rs

```diff
@@ -65,9 +60,6 @@ struct Cli {
     #[clap(long)]
     alarm_topic: Option<String>,
 
-    #[clap(long)]
-    digitiser_event_topic: Option<String>,
-
     #[clap(long)]
     frame_event_topic: Option<String>,
```

Comment:
This should likely no longer be optional.

Reply:
Should any of the topics be optional?